src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -318,13 +354,25 @@ public static class ClientOperations
                 async ctx =>
                 {
                     string content = ctx.GetRandomString(0, ctx.MaxContentLength);
+                    byte[] byteContent = Encoding.ASCII.GetBytes(content);
+                    uint checksum = ChecksumHelpers.Compute(byteContent);
 
-                    using var req = new HttpRequestMessage(HttpMethod.Post, "/duplexSlow") { Content = new ByteAtATimeNoLengthContent(Encoding.ASCII.GetBytes(content)) };
+                    using var req = new HttpRequestMessage(HttpMethod.Post, "/duplexSlow") { Content = new ByteAtATimeNoLengthContent(byteContent) };
                     using HttpResponseMessage m = await ctx.SendAsync(req, HttpCompletionOption.ResponseHeadersRead);
 
-                    ValidateHttpVersion(m, ctx.HttpVersion);
                     ValidateStatusCode(m);
-                    ValidateContent(content, await m.Content.ReadAsStringAsync());
+                    var response = await m.Content.ReadAsStringAsync();
```

Comment:
Style: The expression is not a `new`, `(cast)` or `as cast`, so `var` is not obvious (and thus is not permitted) (rule 10).